.editorconfig

```diff
@@ -8,7 +8,7 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-[{package.json,.travis.yml}]
+[{package.json,.travis.yml,composer.json}]
```

Comment:
How about `[{*.json,.travis.yml}]` so it also covers `.eslintrc.json` and any other `.json` files?